content/get-started/installation/sis.md

```diff
@@ -21,114 +21,138 @@ All you need is an email address! Everything else happens in your 30-day trial a
 
-7. Set your username and password. This will be an admin user account within your Snowflake account. Your Snowflake account can have multiple users within it.
+1. Set your username and password. This will be an admin user account within your Snowflake account. Your Snowflake account can have multiple users within it.
 
-8. If you are not signed in after setting your password, follow the instructions to enter your Snowflake account identifier before entering your username and password. If you've accidentally closed your browser, you can log in at [app.snowflake.com](https://app.snowflake.com/).
+1. If you are not signed in after setting your password, follow the instructions to enter your Snowflake account identifier, then enter your username and password. If you've accidentally closed your browser, you can log in at [app.snowflake.com](https://app.snowflake.com/).
 
-9. Congratulations! You have a trial Snowflake account. The first thing you will see are some sample databases. (This interface is called Snowsight. Check out the Snowflake docs for a [quick tour](https://docs.snowflake.com/en/user-guide/ui-snowsight-quick-tour).)
+Congratulations! You have a trial Snowflake account.
 
-   ![Sample databases in your new trial Snowflake account](/images/get-started/SiS-1-landing-page.png)
+The displayed interface is called Snowsight. Snowsight provides a web-based, graphical user interface for your Snowflake account. The default page is "**Home**," which provides popular quick actions to get started. You can access your "**Projects**" in the left navigation or at the bottom of your "**Home**" page. "**Projects**" include worksheets, notebooks, Streamlit apps, and dashboards. Check out the Snowflake docs for a [quick tour](https://docs.snowflake.com/en/user-guide/ui-snowsight-quick-tour).)
 
-## Accept the use of Anaconda on your Snowflake account
+![Sample databases in your new trial Snowflake account](/images/get-started/SiS-1-landing-page.png)
 
-10. In the left navigation, go to "**Admin**" then "**Billing & Terms**."
+## Optional: Create a warehouse
 
-11. In the "**Anaconda**" section, click "**Enable**."
+Warehouses provide compute resources for tasks and apps in your Snowflake account. Your trial account already has an XS warehouse which you can use. This is named "COMPUTE_WH." However, if you want to use more compute resources, you can create another warehouse.
 
-12. Accept the terms. The "**Anaconda**" section will now show "**Acknowledged**."
+1. In the lower-left corner under your name, confirm that your current role is "ACCOUNTADMIN." If it isn't, click your name, hover over "**Switch Role**," and select "**ACCOUNTADMIN**."
 
-## Create a warehouse and database to hold your Streamlit apps
+1. In the left navigation, select "**Admin**" → "**Warehouses**."
```

Comment:
I recommend > instead of the tiny arrow. Just be consistent.

Reply:
It renders prettier on the site and isn't so tiny.